### DISCUSSION TITLE
Freezing or crashing docker

### DISCUSSION THREAD

**Discussion by ramboton** [mrlt8/docker-wyze-bridge#705] [General]
First off I love this project and I have done a lot to try and make this work, but seriously it is driving me crazy.  Is anyone able to run this for more than 24 hours without having to re-start docker?  So that is my issue, somewhere around 10-12 hours or so the  cameras stop refreshing. I can re-start the container, this does not resolve it. If I try to re-start docker it just locks up and says it is shutting down, I either have to run the task manager and kill the processes or re-boot the computer.

My first installation was on a I5, 16gigs of ram running Windows 10. I was also running another container and that computer was pretty busy running plex and some other stuff so I thought that was the issue.

Next I realized that running a dozen cameras is a lot of work load, so I made it so that it is only pulling 4 cameras, I set docker up on two computers both running windows 10, one is a I5, 16gigs of ram viewing 4 cameras, another one is a I7 8gigs of ram, viewing 4 other cameras.  I am running Debian on windows.

No matter what I do it works great for 10-12 hours or so then the page stops refreshing the cameras and have to kill docker or reboot the computer. both systems are running the most current docker and the most current wyze-bridge. I have seen this issue for the last few revisions and was just hoping that an update to one or the other would resolve it.  Also I am not viewing the cameras for 10-12 hours continuously, I may view them for 10-15 minutes, then close the window, maybe come back an hour view them, close the window and come back later until it stops working.

Is it this bad for everyone or is there something I can do to resolve this?

Thank you

**Comment by mrlt8** [comment]
I believe there was a networking bug introduced in docker desktop (macOS/windows) starting around v4.13 that is causing the bridge to crash. #649

Rolling back to docker desktop v4.12 seems to help. https://docs.docker.com/desktop/release-notes/#4120

It also seems like the issue may have been resolved in docker desktop v4.16 but I haven't had a chance to test it yet.

**Reply by ramboton** [reply]
Thank you I will try that.

**Reply by ramboton** [reply]
Almost 24 hours later and still working great, Thank you.
